lib/src/duration/duration.dart

```diff
@@ -66,14 +67,14 @@ class DurationParsingUtil {
           chronoUnit = ChronoUnit.SECONDS;
           break;
         case DateTimeConstants.TimexDay:
-          chronoUnit = ChronoUnit.DAYS;
+          result = result.AddDays((number * futureOrPast).round());
```

Comment:
I want to confirm that you definitely intended to change a constant into a calculation?

Reply:
We still perform a calculation after assigning the value to `chronoUnit`.

I found an issue in one of the parser tests, where the dates before and after shifting have different time offsets, resulting in different days from what is expected. For example:

Expected date: 2024-02-20 00:00:00
The method returns: 2024-02-19 23:00:00

Because of that I needed to change the calculation to take into account the difference between the time offsets.